python/paddle/fluid/tests/unittests/mkldnn/test_pool2d_bf16_mkldnn_op.py

```diff
@@ -95,6 +136,55 @@ def init_pool_type(self):
         self.pool2D_forward_naive = max_pool2D_forward_naive
 
 
+class TestCase1AvgGradMixin(TestPoolBf16MklDNNOpGradMixin):
```

Comment:
What do you think about changing this test name, so it will include more information about what does it test? "Case1" does not describe what is it testing, what do you think about including more information, f.e. kernel sizes or strides used? This comment applies to every test with "CaseX" in name

Reply:
It's a good idea, however I followed the naming that comes from `test_pool2d_op` tests that are implemented here